lib/mobility/plugins/active_model/dirty.rb

```diff
@@ -142,7 +142,8 @@ def #{method_name}(attr_name, *rest#{kwargs})
                     mutations_from_mobility.attribute_previously_changed?(attr_name))
                   mutations_from_mobility.send(#{method_name.inspect}, attr_name, *rest#{kwargs})
                 else
-                  super
+                  options = rest.first || {}
+                  super(attr_name, **options)
```

Comment:
Hmm, this doesn't look right. There is already some code handling keyword arguments above. Are you sure this fixes the issue? Please add a failing test.